conferences/models.py

```diff
@@ -123,8 +132,10 @@ def __str__(self):
 class UserPreferences(models.Model):
     user = models.ForeignKey(User)
     zosia = models.ForeignKey(Zosia)
-    organization = models.ForeignKey(Organization, null=True, blank=True)
-    bus = models.ForeignKey(Bus, null=True, blank=True)
+    organization = models.ForeignKey(Organization, null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
i think organization should be required - everybody wants to know where are you from :P